### PR TITLE
Support 8.0.1.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
Needs this first https://github.com/ambiata/anatomy-ambiata/pull/348

### CODE REVIEW

File: src/P/Show.hs [hunk 1]
```diff
@@ -6,10 +7,10 @@ module P.Show (
 
 import          Data.Text (Text, pack)
 
-renderIntegral :: (Show a, Integral a) => a -> Text
+renderIntegral :: (Integral a, Show a) => a -> Text
 renderIntegral = pack . show
 
-renderFractional :: (Show a, Fractional a) => a -> Text
+renderFractional :: (Fractional a, Show a) => a -> Text
```
Comment (charleso) [inline src/P/Show.hs]: Any reason for this change?

Comment (tmcgilchrist) [inline src/P/Show.hs]: No, I'll swap them back.

File: ambiata-p.cabal [hunk 2]
```diff
@@ -21,6 +21,11 @@ library
                         , text                        >= 1.1        && < 1.3
                         , transformers                >= 0.4        && < 0.6
 
+  ghc-options: -Wall
+  if impl(ghc >= 8.0)
+    ghc-options: -fno-warn-redundant-constraints
+    
+
   ghc-options:
                        -Wall
```
Comment (jystic) [inline ambiata-p.cabal]: This probably shouldn't be duplicated

File: src/P/Show.hs [hunk 3]
```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE ScopedTypeVariables #-}
```
Comment (jystic) [inline src/P/Show.hs]: Any reason for this?

**Comment by jystic** [comment]
📫

**Comment by erikd-ambiata** [comment]
lgtm

**Comment by tmcgilchrist** [comment]
Note the bump of aeson to <= 12

**Pull request merged**